palm/plugins/dbt/commands/cmd_seed.py

```diff
@@ -0,0 +1,42 @@
+import click
+from typing import Optional, Tuple
+from palm.plugins.dbt.dbt_palm_utils import dbt_env_vars
+
+
+@click.command('seed')
+@click.option(
+    "--clean", is_flag=True, help="drop the test schema after the run is complete"
+)
+@click.option("--select", '-s', multiple=True, help="see dbt docs on select flag")
+@click.option("--exclude", '-e', multiple=True, help="see dbt docs on exclude flag")
+@click.option(
+    "--no-full-refresh",
+    "-nf",
+    is_flag=True,
+    help="Insert seeds instead of recreating the table",
+)
```

Comment:
nit: I would choose one of the three formats 😅

Reply:
PEP8 disagrees. This was formatted by the linter (black)